lib/tasks/eu_exit_business_finder.rake

```diff
@@ -0,0 +1,134 @@
+namespace :eu_exit_business_finder do
+  desc <<-DESC
+    Updates all documents that are tagged with a facet_value with a replacement value
+  DESC
+  task replace_facet_values: [:environment] do
+    facet_values_merges = [
+      {
+        "old_facet_value": "9f3476e1-8ff0-455d-a14e-003236b2797c",
+        "new_facet_value": "53f9ce4c-7cbb-447f-bdf1-a9b022896d3a"
+      },
+
+      {
+        "old_facet_value": "b4e507df-f067-4749-9468-3de120775216",
+        "new_facet_value": "24fd50fa-6619-46ca-96cd-8ce90fa076ce"
+      },
+
+      {
+        "old_facet_value": "1e3e8abd-135d-4844-afa8-5c51df3d3c57",
+        "new_facet_value": "34a6edd0-46ea-4a76-ae80-8c96709d4f59"
+      },
+
+      {
+        "old_facet_value": "c1d8057c-76bf-431c-9ac8-6281a4b7b9ca",
+        "new_facet_value": "34a6edd0-46ea-4a76-ae80-8c96709d4f59"
+      },
+
+      {
+        "old_facet_value": "356f46a0-17d3-4ba4-8952-8c02244f9045",
+        "new_facet_value": "18c3892e-8a0e-4884-906e-5938380eceee"
+      }
+    ]
+
+    content_changes = []
+    facet_values_merges.each do |facet_values_merge|
+      content_ids = []
+      content_links = Services.rummager.search_enum(
+        filter_facet_values: facet_values_merge[:old_facet_value]
+      ).pluck("link")
+      content_links.each do |content_link|
+        content_ids << Services.rummager.get_content(content_link).to_hash['raw_source']['content_id']
+      end
+      content_changes << {
+        "content_ids": content_ids,
+        "old_facet_value": facet_values_merge[:old_facet_value],
+        "new_facet_value": facet_values_merge[:new_facet_value]
+      }
+    end
+
+    bulk_retag_documents(content_changes)
+  end
+
+  desc <<-DESC
+    Removes a specified facet_value from a list of documents, then replaces this with a new facet_value
+  DESC
+  task retag_content_facet_values: [:environment] do
+    content_changes = [
+      {
+        "content_ids": [
+          "02a7bf25-bcc9-4e3c-aaf8-b30eda527589",
+          "ddcac9ba-a852-4fe6-9d1a-73c64e6fe591",
+          "b086e251-aae4-4811-8f9b-08086ad355e2",
+          "6cf648b4-fe85-4eb9-9468-59cd16309475",
+          "a115dfb1-43c7-4a07-9300-87c50187c7c1",
+          "6a8907cc-8200-4df7-877c-2770370c2b0b",
+          "474e8df7-0229-4e96-bf5e-e430f3a19bf2",
+          "4cc71bfd-4a2b-4bb0-b3b2-de9a9909ab53",
+          "9de0b81c-af80-4cf7-827a-e196d6d3811f"
```

Comment:
Could this task be avoided by just adding these content_ids to the `facet_values_merges` array in the first task?

Reply:
I've reworked this too